terraform/lib/dependabot/terraform/file_updater.rb

```diff
@@ -21,10 +22,18 @@ def updated_dependency_files
           next unless file_changed?(file)
 
           updated_content = updated_terraform_file_content(file)
+
           raise "Content didn't change!" if updated_content == file.content
 
           updated_files << updated_file(file: file, content: updated_content)
         end
+        updated_lockfile_content = update_lockfile_declaration
+
```

Comment:
small style nit, would usually leave a empty line after `end` and remove the empty line here
```suggestion
        
        updated_lockfile_content = update_lockfile_declaration
```

Reply:
Rubocop did not like this